docs/FingerprintApi.md

```diff
@@ -147,13 +147,14 @@ Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
  **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
   **limit** | **int32**| Limit the number of events returned.  | 
- **optional** | ***FingerprintApiSearchEventsOpts** | optional parameters | nil if no parameters
+ **opts** | ***FingerprintApiSearchEventsOpts** | optional parameters | nil if no parameters
 
 ### Optional Parameters
 Optional parameters are passed through a pointer to a FingerprintApiSearchEventsOpts struct
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
+**optional** | ***FingerprintApiSearchEventsOpts** | optional parameters | nil if no parameters
```

Comment:
This should be named as `opts` instead of `optional`

Reply:
Fixed, thanks!